extension/onboarding/onboardingView.jsx

```diff
@@ -144,8 +193,9 @@ const OptinAudioDescription = () => {
         collection.
       </p>
       <p className="warning">
-        The microphone is only active when triggered with a button press or
-        keyboard shortcut.
+        {wakewordActive
+          ? 'The microphone is only active when triggered using "Hey Firefox", a button press or keyboard shortcut.'
```

Comment:
Well... this isn't really true, with wakewordActive the microphone will always be on, but won't be sent to any server.

@awallin do you have a text suggestion here? Maybe "With Hey Firefox active the microphone will be listening, but audio will not be sent to a server until after you say Hey Firefox or activate with a button press or keyboard shortcut."